core/src/main/java/com/github/mizool/core/validation/dto/PostDto.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2017 incub8 Software Labs GmbH
+ * Copyright 2017 protel Hotelsoftware GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.mizool.core.validation.dto;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Target({ FIELD, PARAMETER })
```

Comment:
Note that `ElementType.FIELD` is allowed to support injecting dtos into fields. I was first confused as this _also_ allows to annotate sub-dtos inside other dtos, which doesn't make much sense.